lib/utils/power_level_manager.dart

```diff
@@ -14,13 +14,15 @@ class PowerLevelManager {
     return DefaultPowerLevelMember.user.powerLevel;
   }
 
+  int getAdminPowerLevel() => DefaultPowerLevelMember.admin.powerLevel;
+
   Map<String, dynamic> getDefaultPowerLevelEventForMember() {
     return {
       EventTypes.RoomPinnedEvents: getUserPowerLevel(),
-      EventTypes.RoomName: getUserPowerLevel(),
-      EventTypes.RoomAvatar: getUserPowerLevel(),
+      EventTypes.RoomName: getAdminPowerLevel(),
+      EventTypes.RoomAvatar: getAdminPowerLevel(),
       EventTypes.RoomMember: getUserPowerLevel(),
```

Comment:
What is `room Member` event type? I think it relates to `invitation`, `kick`, `ban`? Is it?

Reply:
This event contains membership changes